haskellPackages: update stackage and hackage

sternenseemann opened the pull request:
### This Merge

This PR is the regular merge of the `haskell-updates` branch into `master`.

This branch is being continually built and tested by hydra at https://hydra.nixos.org/jobset/nixpkgs/haskell-updates.

We roughly aim to merge these `haskell-updates` PRs at least once every two weeks. See the @NixOS/haskell [team calendar](https://cloud.maralorn.de/apps/calendar/p/Mw5WLnzsP7fC4Zky) for who is currently in charge of this branch.

### haskellPackages Workflow Summary

Our workflow is currently described in [`pkgs/development/haskell-modules/HACKING.md`](https://github.com/NixOS/nixpkgs/blob/haskell-updates/pkgs/development/haskell-modules/HACKING.md).

The short version is this:
* We regularly update the Stackage and Hackage pins on `haskell-updates` (normally at the beginning of a merge window).
* The community fixes builds of Haskell packages on that branch.
* We aim at at least one merge of `haskell-updates` into `master` every two weeks.
* We only do the merge if the [`mergeable`](https://hydra.nixos.org/job/nixpkgs/haskell-updates/mergeable) job is succeeding on hydra.
* If a [`maintained`](https://hydra.nixos.org/job/nixpkgs/haskell-updates/maintained) package is still broken at the time of merge, we will only merge if the maintainer has been pinged 7 days in advance. (If you care about a Haskell package, become a maintainer!)

---

This is the follow-up to #138596. Come to [#haskell:nixos.org](https://matrix.to/#/#haskell:nixos.org) if you have any questions.

DamienCassou committed:
hledger-check-fancyassertions: init at 1.23

Co-authored-by: sterni <sternenseemann@systemli.org>

sternenseemann committed:
all-cabal-hashes: 2021-09-19T21:23:33Z -> 2021-09-29T20:58:23Z

This commit has been generated by maintainers/scripts/haskell/update-hackage.sh

sternenseemann committed:
haskellPackages: stackage-lts 18.10 -> 18.12

This commit has been generated by maintainers/scripts/haskell/update-stackage.sh

sternenseemann committed:
haskellPackages: regenerate package set based on current config

This commit has been generated by maintainers/scripts/haskell/regenerate-hackage-packages.sh

sternenseemann committed:
haskell.packages.ghc901.ormolu: 0.3.0.0 -> 0.3.0.1

Fixes evaluation of ghc901 package set.

github-actions[bot] committed:
Merge master into haskell-updates

maralorn committed:
haskellPackages.hls-call-hierarchy-plugin: dontCheck on darwin

sternenseemann committed:
haskellPackages.hledger-lib_1_23: build with doctest 0.18.1

Test suite has a lower bound on doctest >= 0.18.

maralorn committed:
Merge pull request #139534 from DamienCassou/hledger-check-fancyassertions

hledger-check-fancyassertions: init at 1.23

sternenseemann commented:
### [haskell-updates build report from hydra](https://hydra.nixos.org/jobset/nixpkgs/haskell-updates)
*evaluation [1709405](https://hydra.nixos.org/eval/1709405) of nixpkgs commit [0365ec6](https://github.com/NixOS/nixpkgs/commits/0365ec67aff35db0e03e73e59f6c5fa974763358) as of 2021-09-30 18:07 UTC*

:red_circle: **Branch not mergeable**
  * `mergeable` jobset failed.
  * `maintained` jobset failed.

#### Build summary

 | Platform | Failed :x: | DependencyFailed :heavy_exclamation_mark: | TimedOut :hourglass::no_entry_sign: | Unfinished :hourglass_flowing_sand: | Success :heavy_check_mark: | 
 | --- | --- | --- | --- | --- | --- | 
 | [aarch64-linux :iphone:](https://hydra.nixos.org/eval/1709405?filter=.aarch64-linux) | 26 | 14 | 4 |  | 6858 | 
 | [x86_64-darwin :apple:](https://hydra.nixos.org/eval/1709405?filter=.x86_64-darwin) | 56 | 34 | 12 | 69 | 6684 | 
 | [x86_64-linux :penguin:](https://hydra.nixos.org/eval/1709405?filter=.x86_64-linux) | 10 | 9 | 1 |  | 6928 | 
#### Maintained packages with build failure
- [ ] [nvfetcher](https://hydra.nixos.org/eval/1709405?filter=nvfetcher) @berberman
  - [[:iphone::x:]](https://hydra.nixos.org/build/154940466) [[:apple::x:]](https://hydra.nixos.org/build/154938040) [[:penguin::x:]](https://hydra.nixos.org/build/154934862) [toplevel](https://hydra.nixos.org/eval/1709405?filter=nvfetcher)
  - [[:iphone::x:]](https://hydra.nixos.org/build/154940153) [[:apple::x:]](https://hydra.nixos.org/build/154934287) [[:penguin::x:]](https://hydra.nixos.org/build/154936583) [haskellPackages](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.nvfetcher)
#### Maintained packages with failed dependency
- [ ] [haskell-language-server](https://hydra.nixos.org/eval/1709405?filter=haskell-language-server) @maralorn
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154940784) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/155010233) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154938728) [toplevel](https://hydra.nixos.org/eval/1709405?filter=haskell-language-server)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154934325) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/155010240) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154939184) [haskell.packages.ghc8107](https://hydra.nixos.org/eval/1709405?filter=haskell.packages.ghc8107.haskell-language-server)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154934776) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/155010236) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154940429) [haskell.packages.ghc884](https://hydra.nixos.org/eval/1709405?filter=haskell.packages.ghc884.haskell-language-server)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154933892) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/155010239) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154940778) [haskell.packages.ghc901](https://hydra.nixos.org/eval/1709405?filter=haskell.packages.ghc901.haskell-language-server)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154935466) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/155010235) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154939981) [haskellPackages](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.haskell-language-server)
- [ ] [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/155010238) [maintained](https://hydra.nixos.org/eval/1709405?filter=maintained) @cdepillabout @expipiplus1 @maralorn @sternenseemann
- [ ] [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/155010234) [mergeable](https://hydra.nixos.org/eval/1709405?filter=mergeable) @cdepillabout @expipiplus1 @maralorn @sternenseemann
#### Unmaintained packages with build failure
<details><summary>77 job(s) </summary>

- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154789224) [[:apple::x:]](https://hydra.nixos.org/build/154785808) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154785575) [haskellPackages.sdp](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.sdp)  :arrow_heading_up: 9 | 9
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154772259) [[:apple::x:]](https://hydra.nixos.org/build/154780267) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154775947) [haskellPackages.di-core](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.di-core)  :arrow_heading_up: 7 | 11
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154937117) [[:apple::x:]](https://hydra.nixos.org/build/154932436) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154940168) [haskellPackages.thyme](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.thyme)  :arrow_heading_up: 6 | 15
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/154789698) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/154784464) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154776425) [haskellPackages.libBF](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.libBF)  :arrow_heading_up: 4 | 20
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154938582) [[:apple::x:]](https://hydra.nixos.org/build/154939341) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154940573) [haskellPackages.exinst](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.exinst)  :arrow_heading_up: 4 | 6
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/154936544) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/154934103) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154940975) [haskellPackages.ptr-poker](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.ptr-poker)  :arrow_heading_up: 3 | 3
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154935355) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/154937501) [[:penguin::x:]](https://hydra.nixos.org/build/154933657) [haskellPackages.factory](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.factory)  :arrow_heading_up: 2 | 4
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/154933588) [[:apple::x:]](https://hydra.nixos.org/build/154937808) [[:penguin::x:]](https://hydra.nixos.org/build/154940759) [haskellPackages.quic](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.quic)  :arrow_heading_up: 2 | 2
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/154935861) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/154936025) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154938502) [haskellPackages.OrderedBits](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.OrderedBits)  :arrow_heading_up: 1 | 36
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/154934538) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/154938551) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154941374) [haskellPackages.type-natural](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.type-natural)  :arrow_heading_up: 1 | 4
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/154778194) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/154792315) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154781151) [haskellPackages.long-double](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.long-double)  :arrow_heading_up: 1 | 2
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/154781560) [[:apple::x:]](https://hydra.nixos.org/build/154776628) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154784127) [haskellPackages.easytensor](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.easytensor)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154931913) [[:apple::x:]](https://hydra.nixos.org/build/154939171) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154941063) [haskellPackages.gi-gdkx11](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.gi-gdkx11)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154935594) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/154937051) [[:penguin::x:]](https://hydra.nixos.org/build/154936276) [haskellPackages.hls-module-name-plugin](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.hls-module-name-plugin)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154780414) [[:apple::x:]](https://hydra.nixos.org/build/154778580) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154786630) [haskellPackages.keep-alive](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.keep-alive)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154782681) [[:apple::x:]](https://hydra.nixos.org/build/154775174) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154780043) [haskellPackages.loc](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.loc)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/154935986) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/154934424) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154938447) [haskellPackages.mighty-metropolis](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.mighty-metropolis)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/154933556) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/154932492) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154933768) [haskellPackages.nlopt-haskell](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.nlopt-haskell)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154937041) [[:apple::x:]](https://hydra.nixos.org/build/154932752) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154936048) [haskellPackages.opencv](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.opencv)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/154773178) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/154784307) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154774332) [haskellPackages.unicode-properties](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.unicode-properties)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/154938846) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/154938624) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154934326) [haskellPackages.accelerate-llvm](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.accelerate-llvm)  :arrow_heading_up: 0 | 8
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/154787396) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/154782067) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154779155) [haskellPackages.freetype2](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.freetype2)  :arrow_heading_up: 0 | 7
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154939463) [[:apple::x:]](https://hydra.nixos.org/build/154939305) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154939338) [haskellPackages.pipes-zlib](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.pipes-zlib)  :arrow_heading_up: 0 | 6
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154784823) [[:apple::x:]](https://hydra.nixos.org/build/154780359) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154771786) [haskellPackages.hmidi](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.hmidi)  :arrow_heading_up: 0 | 4
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154932503) [[:apple::x:]](https://hydra.nixos.org/build/154933237) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154931958) [haskellPackages.zip](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.zip)  :arrow_heading_up: 0 | 4
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/154776274) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/154792401) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154784925) [haskellPackages.cdar-mBound](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.cdar-mBound)  :arrow_heading_up: 0 | 2
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154934500) [[:apple::x:]](https://hydra.nixos.org/build/154939367) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154941031) [haskellPackages.posix-socket](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.posix-socket)  :arrow_heading_up: 0 | 2
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154785547) [[:apple::x:]](https://hydra.nixos.org/build/154781131) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154786703) [haskellPackages.hamid](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.hamid)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154939644) [[:apple::x:]](https://hydra.nixos.org/build/154936331) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154936367) [haskellPackages.hmatrix-morpheus](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.hmatrix-morpheus)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154783677) [[:apple::x:]](https://hydra.nixos.org/build/154786798) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154784002) [haskellPackages.huckleberry](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.huckleberry)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/154777045) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/154787604) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154776904) [haskellPackages.picosat](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.picosat)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154778698) [[:apple::x:]](https://hydra.nixos.org/build/154787700) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154792170) [haskellPackages.select](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.select)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154785987) [[:apple::x:]](https://hydra.nixos.org/build/154792412) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154790024) [haskellPackages.sysinfo](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.sysinfo)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154934027) [[:apple::x:]](https://hydra.nixos.org/build/154933158) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154933284) [haskellPackages.FractalArt](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.FractalArt) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/154789939) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/154773319) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154788245) [haskellPackages.HsASA](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.HsASA) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154935132) [[:apple::x:]](https://hydra.nixos.org/build/154933282) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154939328) [haskellPackages.broadcast-chan-conduit](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.broadcast-chan-conduit) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/154936792) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/154941067) [[:penguin::x:]](https://hydra.nixos.org/build/154932739) [haskellPackages.candid](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.candid) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154779690) [[:apple::x:]](https://hydra.nixos.org/build/154780780) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154786075) [haskellPackages.chiphunk](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.chiphunk) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154773174) [[:apple::x:]](https://hydra.nixos.org/build/154791571) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154775226) [haskellPackages.discount](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.discount) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154787540) [[:apple::x:]](https://hydra.nixos.org/build/154779945) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154783993) [haskellPackages.diskhash](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.diskhash) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154939952) [[:apple::hourglass::no_entry_sign:]](https://hydra.nixos.org/build/154931944) [[:penguin::x:]](https://hydra.nixos.org/build/154937667) [haskellPackages.docusign-base](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.docusign-base) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154781921) [[:apple::x:]](https://hydra.nixos.org/build/154771661) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154778464) [haskellPackages.epub-tools](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.epub-tools) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154791888) [[:apple::x:]](https://hydra.nixos.org/build/154786682) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154773297) [haskellPackages.float128](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.float128) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/154778113) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154785100) [haskellPackages.gnome-keyring](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.gnome-keyring) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154934547) [[:apple::x:]](https://hydra.nixos.org/build/154938450) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154934148) [haskellPackages.gtk-traymanager](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.gtk-traymanager) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154776219) [[:apple::x:]](https://hydra.nixos.org/build/154782580) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154779257) [haskellPackages.hid](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.hid) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154935912) [[:apple::x:]](https://hydra.nixos.org/build/154935892) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154934339) [haskellPackages.highlight](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.highlight) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/154937777) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/154939281) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154940607) [haskellPackages.hq](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.hq) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154939088) [[:apple::x:]](https://hydra.nixos.org/build/154935318) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154937489) [haskellPackages.hs](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.hs) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154792168) [[:apple::x:]](https://hydra.nixos.org/build/154772744) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154771981) [haskellPackages.hsshellscript](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.hsshellscript) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154780793) [[:apple::x:]](https://hydra.nixos.org/build/154778712) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154781385) [haskellPackages.hssourceinfo](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.hssourceinfo) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154781557) [[:apple::x:]](https://hydra.nixos.org/build/154778553) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154773264) [haskellPackages.ipcvar](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.ipcvar) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154786688) [[:apple::x:]](https://hydra.nixos.org/build/154791949) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154780470) [haskellPackages.linux-framebuffer](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.linux-framebuffer) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154936952) [[:apple::x:]](https://hydra.nixos.org/build/154938433) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154940516) [haskellPackages.mediawiki2latex](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.mediawiki2latex) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154792910) [[:apple::x:]](https://hydra.nixos.org/build/154773939) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154779615) [haskellPackages.mercury-api](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.mercury-api) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154792505) [[:apple::x:]](https://hydra.nixos.org/build/154782578) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154782796) [haskellPackages.nano-cryptr](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.nano-cryptr) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154940480) [[:apple::x:]](https://hydra.nixos.org/build/154932656) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154937388) [haskellPackages.nri-http](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.nri-http) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154932117) [[:apple::x:]](https://hydra.nixos.org/build/154932372) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154940074) [haskellPackages.persistent-pagination](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.persistent-pagination) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154936405) [[:apple::x:]](https://hydra.nixos.org/build/154933060) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154932759) [haskellPackages.ping-wrapper](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.ping-wrapper) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/154940523) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/154939535) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154940449) [haskellPackages.poker](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.poker) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154774836) [[:apple::x:]](https://hydra.nixos.org/build/154791346) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154784566) [haskellPackages.posix-timer](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.posix-timer) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154784792) [[:apple::x:]](https://hydra.nixos.org/build/154784840) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154775780) [haskellPackages.pthread](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.pthread) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154940094) [[:apple::x:]](https://hydra.nixos.org/build/154933013) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154933653) [haskellPackages.sandwich-webdriver](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.sandwich-webdriver) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/154934071) [[:apple::x:]](https://hydra.nixos.org/build/154941562) [[:penguin::x:]](https://hydra.nixos.org/build/154939054) [haskellPackages.semigroupoids-do](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.semigroupoids-do) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154778592) [[:apple::x:]](https://hydra.nixos.org/build/154789655) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154787708) [haskellPackages.sfml-audio](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.sfml-audio) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154792892) [[:apple::x:]](https://hydra.nixos.org/build/154771791) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154792865) [haskellPackages.shared-memory](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.shared-memory) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/154941449) [[:apple::x:]](https://hydra.nixos.org/build/154940057) [[:penguin::x:]](https://hydra.nixos.org/build/154935928) [haskellPackages.streamly-lz4](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.streamly-lz4) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154935431) [[:apple::x:]](https://hydra.nixos.org/build/154941527) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154937726) [haskellPackages.tailfile-hinotify](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.tailfile-hinotify) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/154937715) [[:apple::x:]](https://hydra.nixos.org/build/154938118) [[:penguin::x:]](https://hydra.nixos.org/build/154936450) [haskellPackages.tmp-proc-example](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.tmp-proc-example) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/154785076) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/154790993) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154789304) [haskellPackages.wiringPi](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.wiringPi) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154784991) [[:apple::x:]](https://hydra.nixos.org/build/154786464) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154782315) [tests.haskell.writers](https://hydra.nixos.org/eval/1709405?filter=tests.haskell.writers) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/154939646) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/154938013) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154938787) [haskellPackages.x86-64bit](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.x86-64bit) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/154940852) [[:apple::x:]](https://hydra.nixos.org/build/154932130) [[:penguin::x:]](https://hydra.nixos.org/build/154940954) [haskellPackages.xml-verify](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.xml-verify) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154789553) [[:apple::x:]](https://hydra.nixos.org/build/154780798) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154786667) [haskellPackages.xmonad-utils](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.xmonad-utils) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154775908) [[:apple::x:]](https://hydra.nixos.org/build/154773446) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154779378) [haskellPackages.yoga](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.yoga) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154792126) [[:apple::x:]](https://hydra.nixos.org/build/154777007) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154785640) [haskellPackages.zot](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.zot) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154789280) [[:apple::x:]](https://hydra.nixos.org/build/154790939) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154779177) [haskellPackages.zxcvbn-c](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.zxcvbn-c) 
</details>

#### Unmaintained packages with failed dependency
<details><summary>45 job(s) </summary>

- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154786462) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154785450) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154789537) [haskellPackages.di-handle](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.di-handle)  :arrow_heading_up: 5 | 9
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154789171) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154778125) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154781342) [haskellPackages.di-monad](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.di-monad)  :arrow_heading_up: 5 | 9
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154938832) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154932289) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154938053) [haskellPackages.di-df1](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.di-df1)  :arrow_heading_up: 4 | 8
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154935248) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/154941371) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154936047) [haskellPackages.jsonifier](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.jsonifier)  :arrow_heading_up: 2 | 2
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154792513) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154772005) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154773444) [haskellPackages.sdp-io](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.sdp-io)  :arrow_heading_up: 2 | 2
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154936130) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154934935) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154939365) [haskellPackages.di-polysemy](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.di-polysemy)  :arrow_heading_up: 1 | 4
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154939878) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154938607) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154936362) [haskellPackages.hbro](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.hbro)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154932233) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154936835) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154937841) [haskellPackages.http3](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.http3)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154937670) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/154933187) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154940657) [haskellPackages.opentelemetry-extra](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.opentelemetry-extra)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154933665) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154939331) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154940731) [haskellPackages.orgmode-parse](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.orgmode-parse)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154783272) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154773863) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154780098) [haskellPackages.sdp-hashable](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.sdp-hashable)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154933115) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/154937092) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154933220) [haskellPackages.PrimitiveArray](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.PrimitiveArray)  :arrow_heading_up: 0 | 35
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154936545) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154937620) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154935829) [haskellPackages.di](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.di)  :arrow_heading_up: 0 | 2
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154932009) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/154939076) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154937363) [haskellPackages.sized](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.sized)  :arrow_heading_up: 0 | 2
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154935076) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154936369) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154938135) [haskellPackages.keenser](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.keenser)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154939775) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154937538) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154934656) [haskellPackages.moto](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.moto)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154933017) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154938904) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154938664) [haskellPackages.antiope-es](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.antiope-es) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154937525) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/154936632) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154940322) [haskellPackages.declarative](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.declarative) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154777577) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154780718) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154788154) [haskellPackages.easytensor-vulkan](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.easytensor-vulkan) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154936815) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154940200) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154934408) [haskellPackages.exinst-aeson](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.exinst-aeson) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154939146) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154931877) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154940209) [haskellPackages.exinst-bytes](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.exinst-bytes) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154939003) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154941075) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154940033) [haskellPackages.exinst-cereal](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.exinst-cereal) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154941584) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154937344) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154939090) [haskellPackages.exinst-serialise](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.exinst-serialise) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154933080) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154937912) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154933381) [haskellPackages.fastparser](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.fastparser) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154933769) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/154940996) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154941115) [haskellPackages.fishfood](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.fishfood) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154941141) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154938883) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154936199) [haskellPackages.hbro-contrib](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.hbro-contrib) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154941126) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/154936599) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154934616) [haskellPackages.hmatrix-nlopt](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.hmatrix-nlopt) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154936727) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154934089) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154937099) [haskellPackages.opencv-extra](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.opencv-extra) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154932272) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/154940294) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154937054) [haskellPackages.opentelemetry-lightstep](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.opentelemetry-lightstep) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154938153) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154934249) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154941291) [haskellPackages.orgstat](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.orgstat) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154937935) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154933309) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154940893) [haskellPackages.polysemy-log-di](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.polysemy-log-di) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154941255) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154934100) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154934075) [haskellPackages.postgresql-replicant](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.postgresql-replicant) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154774635) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/154777385) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154782412) [haskellPackages.rounded](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.rounded) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154934106) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154938108) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154936578) [haskellPackages.scan-metadata](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.scan-metadata) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154774962) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154788261) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154779848) [haskellPackages.sdp-binary](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.sdp-binary) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154774095) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154772632) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154775367) [haskellPackages.sdp-deepseq](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.sdp-deepseq) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154779353) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154773896) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154785343) [haskellPackages.sdp-quickcheck](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.sdp-quickcheck) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154937326) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154935103) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154935366) [haskellPackages.sdp4bytestring](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.sdp4bytestring) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154935327) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154941196) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154936595) [haskellPackages.sdp4text](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.sdp4text) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154771871) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154779023) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154772240) [haskellPackages.sdp4unordered](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.sdp4unordered) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154938779) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154933849) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154934217) [haskellPackages.sdp4vector](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.sdp4vector) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154936683) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/154931874) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154934580) [haskellPackages.squeeze](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.squeeze) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154775944) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/154787766) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154772887) [haskellPackages.unicode-names](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.unicode-names) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154934859) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154934613) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154936816) [haskellPackages.warp-quic](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.warp-quic) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/154787511) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/154777303) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/154773158) [haskellPackages.xbattbar](https://hydra.nixos.org/eval/1709405?filter=haskellPackages.xbattbar) 
</details>

#### Top 50 broken packages, sorted by number of reverse dependencies
<details><summary>50 job(s) </summary>

[gogol-core](https://packdeps.haskellers.com/reverse/gogol-core) :arrow_heading_up: 182  
[haskell98](https://packdeps.haskellers.com/reverse/haskell98) :arrow_heading_up: 153  
[enumerator](https://packdeps.haskellers.com/reverse/enumerator) :arrow_heading_up: 56  
[derive](https://packdeps.haskellers.com/reverse/derive) :arrow_heading_up: 48  
[contiguous](https://packdeps.haskellers.com/reverse/contiguous) :arrow_heading_up: 45  
[MonadCatchIO-transformers](https://packdeps.haskellers.com/reverse/MonadCatchIO-transformers) :arrow_heading_up: 41  
[parseargs](https://packdeps.haskellers.com/reverse/parseargs) :arrow_heading_up: 41  
[bytesmith](https://packdeps.haskellers.com/reverse/bytesmith) :arrow_heading_up: 35  
[data-lens](https://packdeps.haskellers.com/reverse/data-lens) :arrow_heading_up: 34  
[distributed-process](https://packdeps.haskellers.com/reverse/distributed-process) :arrow_heading_up: 30  
[iteratee](https://packdeps.haskellers.com/reverse/iteratee) :arrow_heading_up: 29  
[jmacro](https://packdeps.haskellers.com/reverse/jmacro) :arrow_heading_up: 29  
[ip](https://packdeps.haskellers.com/reverse/ip) :arrow_heading_up: 26  
[either-unwrap](https://packdeps.haskellers.com/reverse/either-unwrap) :arrow_heading_up: 25  
[HList](https://packdeps.haskellers.com/reverse/HList) :arrow_heading_up: 23  
[SciBaseTypes](https://packdeps.haskellers.com/reverse/SciBaseTypes) :arrow_heading_up: 22  
[haskelldb](https://packdeps.haskellers.com/reverse/haskelldb) :arrow_heading_up: 22  
[hsc3](https://packdeps.haskellers.com/reverse/hsc3) :arrow_heading_up: 22  
[wxdirect](https://packdeps.haskellers.com/reverse/wxdirect) :arrow_heading_up: 22  
[BiobaseTypes](https://packdeps.haskellers.com/reverse/BiobaseTypes) :arrow_heading_up: 21  
[wxc](https://packdeps.haskellers.com/reverse/wxc) :arrow_heading_up: 21  
[biocore](https://packdeps.haskellers.com/reverse/biocore) :arrow_heading_up: 20  
[secp256k1-haskell](https://packdeps.haskellers.com/reverse/secp256k1-haskell) :arrow_heading_up: 20  
[wxcore](https://packdeps.haskellers.com/reverse/wxcore) :arrow_heading_up: 20  
[attoparsec-enumerator](https://packdeps.haskellers.com/reverse/attoparsec-enumerator) :arrow_heading_up: 19  
[bytestring-show](https://packdeps.haskellers.com/reverse/bytestring-show) :arrow_heading_up: 19  
[bytestring-trie](https://packdeps.haskellers.com/reverse/bytestring-trie) :arrow_heading_up: 19  
[numhask](https://packdeps.haskellers.com/reverse/numhask) :arrow_heading_up: 19  
[polysemy-plugin](https://packdeps.haskellers.com/reverse/polysemy-plugin) :arrow_heading_up: 19  
[wx](https://packdeps.haskellers.com/reverse/wx) :arrow_heading_up: 19  
[BiobaseENA](https://packdeps.haskellers.com/reverse/BiobaseENA) :arrow_heading_up: 18  
[asn1-data](https://packdeps.haskellers.com/reverse/asn1-data) :arrow_heading_up: 18  
[dbus-core](https://packdeps.haskellers.com/reverse/dbus-core) :arrow_heading_up: 18  
[gtksourceview2](https://packdeps.haskellers.com/reverse/gtksourceview2) :arrow_heading_up: 18  
[BiobaseXNA](https://packdeps.haskellers.com/reverse/BiobaseXNA) :arrow_heading_up: 17  
[HGamer3D-Data](https://packdeps.haskellers.com/reverse/HGamer3D-Data) :arrow_heading_up: 17  
[certificate](https://packdeps.haskellers.com/reverse/certificate) :arrow_heading_up: 17  
[dbus-client](https://packdeps.haskellers.com/reverse/dbus-client) :arrow_heading_up: 17  
[gconf](https://packdeps.haskellers.com/reverse/gconf) :arrow_heading_up: 17  
[gtk-serialized-event](https://packdeps.haskellers.com/reverse/gtk-serialized-event) :arrow_heading_up: 17  
[uuid-orphans](https://packdeps.haskellers.com/reverse/uuid-orphans) :arrow_heading_up: 17  
[cuda](https://packdeps.haskellers.com/reverse/cuda) :arrow_heading_up: 16  
[happstack-jmacro](https://packdeps.haskellers.com/reverse/happstack-jmacro) :arrow_heading_up: 16  
[manatee-core](https://packdeps.haskellers.com/reverse/manatee-core) :arrow_heading_up: 16  
[monads-fd](https://packdeps.haskellers.com/reverse/monads-fd) :arrow_heading_up: 16  
[murmur3](https://packdeps.haskellers.com/reverse/murmur3) :arrow_heading_up: 16  
[tls-extra](https://packdeps.haskellers.com/reverse/tls-extra) :arrow_heading_up: 16  
[ADPfusion](https://packdeps.haskellers.com/reverse/ADPfusion) :arrow_heading_up: 15  
[MaybeT](https://packdeps.haskellers.com/reverse/MaybeT) :arrow_heading_up: 15  
[blaze-builder-enumerator](https://packdeps.haskellers.com/reverse/blaze-builder-enumerator) :arrow_heading_up: 15  
</details>


*:arrow_heading_up:: The number of packages that depend (directly or indirectly) on this package (if any). If two numbers are shown the first (lower) number considers only packages which currently have enabled hydra jobs, i.e. are not marked broken. The second (higher) number considers all packages.*

*Report generated with [maintainers/scripts/haskell/hydra-report.hs](https://github.com/NixOS/nixpkgs/blob/haskell-updates/maintainers/scripts/haskell/hydra-report.sh)*

berberman committed:
haskellPackages.nvfetcher: disable check

github-actions[bot] committed:
Merge master into haskell-updates

sternenseemann commented:
@nomeata Are you interested in becoming a maintainer for `candid`? Seems like it started to fail due to deprecations warnings from a dependency: <https://hydra.nixos.org/build/154932739>

queezle42 committed:
haskell.packages.ghc921: use random_1_2_1

queezle42 committed:
haskell.packages.ghc921.ChasingBottoms: remove markBrokenVersion

queezle42 committed:
haskell.packages.ghc921: re-enable tests previously blocked by 'random'

nomeata commented:
Sure, since I am already upstream, I guess I can't say no here

sternenseemann commented:
You could of course also opt to not care. :) In the current case appending `--ghc-option=-Wno-deprecations` to `configureFlags` should fix the issue if I understand it correctly.

nomeata commented:
I already fixed this since https://github.com/nomeata/haskell-candid/pull/11, but not uploaded to Hackage yet; will do that right away.

nomeata commented:
Done, <https://hackage.haskell.org/package/candid-0.3>. Is there more you’d like me to do?

sternenseemann commented:
> Will this perculate automatically?

No, but I can upgrade it manually or commit a workaround until the next hackage bump, I'll check.

> Is there more you’d like me to do?

If you want to, consider adding yourself and your packages to `pkgs/development/haskell-modules/configuration-hackage2nix/main.yaml` in the `package-mantainers` section. Then you'll get pinged automatically about any breakages and they'll get a second look in general.

nomeata committed:
Add myself as maintainer for a few Haskell packages

as nudged to do so by @sternenseemann in
https://github.com/NixOS/nixpkgs/pull/139967#issuecomment-932186960

sternenseemann committed:
haskellPackages.candid: add workaround for doctest failure

This is fixed properly in candid 0.3 which we'll have soon, but in
the meantime we can just disable the offending warning. A real shame
that doctests are influenced by compiler warnings.

sternenseemann committed:
Merge pull request #140146 from nomeata/joachim/haskell-maintainership

Add myself as maintainer for a few Haskell packages

queezle42 committed:
haskell.packages.ghc921: use network_3_1_2_2

sternenseemann committed:
haskellPackages.candid: 0.2 -> 0.3

Turns out the cabal configure flags don't affect the doctest suite
necessarily, so upgrading candid manually is much easier.

sternenseemann committed:
haskellPackages: mark builds failing on hydra as broken

This commit has been generated by maintainers/scripts/haskell/mark-broken.sh

sternenseemann committed:
haskellPackages.quic: provide correct version of network

sternenseemann committed:
Merge remote-tracking branch 'origin/master' into haskell-updates
